target/linux/brcm47xx/image/Makefile

```diff
@@ -536,146 +587,167 @@ endif
 
 ifeq ($(SUBTARGET),mips74k)
 define Device/asus-rt-ac53u
-  DEVICE_TITLE := Asus RT-AC53U
+  DEVICE_VENDOR := Asus
+  DEVICE_MODEL := RT-AC53U
   DEVICE_PACKAGES := $(USB2_PACKAGES)
   $(Device/asus)
   PRODUCTID := RT-AC53U
 endef
 TARGET_DEVICES += asus-rt-ac53u
 
 define Device/asus-rt-ac66u
-  DEVICE_TITLE := Asus RT-AC66U
+  DEVICE_VENDOR := Asus
+  DEVICE_MODEL := RT-AC66U
   DEVICE_PACKAGES := kmod-b43 $(USB2_PACKAGES)
   $(Device/asus)
   PRODUCTID := RT-AC66U
 endef
 #  TARGET_DEVICES += asus-rt-ac66u
 
 define Device/asus-rt-n10
-  DEVICE_TITLE := Asus RT-N10
+  DEVICE_VENDOR := Asus
+  DEVICE_MODEL := RT-N10
   DEVICE_PACKAGES := kmod-b43
   $(Device/asus)
   PRODUCTID := "RT-N10      "
 endef
 TARGET_DEVICES += asus-rt-n10
 
 define Device/asus-rt-n10p
-  DEVICE_TITLE := Asus RT-N10P
+  DEVICE_VENDOR := Asus
+  DEVICE_MODEL := RT-N10P
   DEVICE_PACKAGES := kmod-b43
   $(Device/asus)
   PRODUCTID := RT-N10P
 endef
 TARGET_DEVICES += asus-rt-n10p
 
 define Device/asus-rt-n10p-v2
-  DEVICE_TITLE := Asus RT-N10P v2
+  DEVICE_VENDOR := Asus
+  DEVICE_MODEL := RT-N10P
+  DEVICE_VARIANT := v2
   $(Device/asus)
   PRODUCTID := RT-N10PV2
 endef
 TARGET_DEVICES += asus-rt-n10p-v2
 
 define Device/asus-rt-n10u
-  DEVICE_TITLE := Asus RT-N10U
+  DEVICE_VENDOR := Asus
+  DEVICE_MODEL := RT-N10U
   DEVICE_PACKAGES := kmod-b43 $(USB2_PACKAGES)
   $(Device/asus)
   PRODUCTID := RT-N10U
 endef
 TARGET_DEVICES += asus-rt-n10u
 
 define Device/asus-rt-n10u-b
-  DEVICE_TITLE := Asus RT-N10U B
+  DEVICE_VENDOR := Asus
+  DEVICE_MODEL := RT-N10U B
   DEVICE_PACKAGES := kmod-b43 $(USB2_PACKAGES)
   $(Device/asus)
   PRODUCTID := RT-N10U
 endef
 TARGET_DEVICES += asus-rt-n10u-b
 
 define Device/asus-rt-n12
-  DEVICE_TITLE := Asus RT-N12
+  DEVICE_VENDOR := Asus
+  DEVICE_MODEL := RT-N12
   DEVICE_PACKAGES := kmod-b43
   $(Device/asus)
   PRODUCTID := "RT-N12      "
 endef
 TARGET_DEVICES += asus-rt-n12
 
 define Device/asus-rt-n12-b1
-  DEVICE_TITLE := Asus RT-N12 B1
+  DEVICE_VENDOR := Asus
+  DEVICE_MODEL := RT-N12 B1
```

Comment:
B1 = variant? Previous device?

Reply:
"B1" or "b1"?

Reply:
I think those were always with capital letters, also on ramips. I would stick to that.

Reply:
ok